docs/trustedsetup.md

```diff
@@ -10,28 +10,14 @@ to manage a large amount of value, and if their users accept that the risk of
 coordinator misbehaviour is insufficient to justify doing the work nof a
 trusted setup. After all, MACI's security model presumes a trusted coordinator.
 
+The PSE team is making available a trusted set of zKeys for MACI's circuits,
+which will be available as soon as the first ceremony is completed and
```

Comment:
```suggestion
The PSE team is in the process of creating a trusted setup for MACI's circuits. The public parameters will be available as soon as the first ceremony is completed and
```

I'd recommend clarifying the terminology for better technical accuracy. Instead of "trusted set," the more standard term is "trusted setup." If you're referring to the output of this setup, it might be better to use "public parameters."

Reply:
a set of zKeys here simply means 2+ zKeys, I could have used the word pair in this case probably (one for process messages and one for tally votes). Anyways, the maci maci ceremony is nown ended, so might as well change the wording of this doc and add links to the keys 😂

Reply:
ahh I see XD